api/app/src/external/commonwell/document/document-query.ts

```diff
@@ -203,6 +206,63 @@ async function internalGetDocuments({
   return documents;
 }
 
+function reportCWErrors({
+  errors,
+  context,
+  log,
+}: {
+  errors: OperationOutcome[];
+  context: Record<string, unknown>;
+  log: ReturnType<typeof Util.out>["log"];
+}): void {
+  const errorsByCategory = groupCWErrors(errors);
+  for (const [category, errors] of Object.entries(errorsByCategory)) {
+    const msg = `Document query error - ${category}`;
+    log(`${msg}: ${JSON.stringify(errors)}`);
```

Comment:
nit: `stringify(errors, null, 2)` for nicer formatting?

Reply:
avoiding these until we [fix multiline logs on CloudWatch](https://github.com/metriport/metriport-internal/issues/693) - currently these multiline logs make it much harder to read the logs